# return the absolute angular from leobase's position （x , y ）, yaw indicate yaw
float32 x
float32 y
float32 yaw
